src/airbash

```diff
@@ -8,41 +7,79 @@
 # access point, thus forcing a reconnect. In order to capture the handshake,
 # airodump is being run after sending the deauthentication packet. Verification
 # of a captured handshake is done using aircrack-ng
+
+# get platform
+if [ -d "/system" ]; then
+    device=0  # android
+else
+    device=1  # linux
+fi
+
 airocheck() {
 	# checks a handshake (passed by file name) using aircrack-ng
 	# returns 1 on success and 0 on failure
 	for s in `ls "$1"*.cap 2>/dev/null`; do
-		if [ `aircrack-ng -a 2 "$s" -w "$path$wl".c 2>/dev/null | grep -c valid` -eq 0 ]; then
-			if [ `aircrack-ng -a 2 "$s" -w "$path$wl".c 2>/dev/null | awk '{print $NF}' | grep -c handshake` -ne 0 ]; then
+		if [ `"$AIRCRACK_BIN" -a 2 "$s" -w "$path$wl"known.txt 2>/dev/null | grep -c valid` -eq 0 ]; then
```

Comment:
As known.txt now contains sample passwords (nice idea :)), we don't need a dummy file anymore to check for valid handshakes.